### ISSUE TITLE
LaneNet Accuracy 

### ISSUE THREAD

**Issue by LukasProgram** [MaybeShewill-CV/lanenet-lane-detection#269]
Hi, I have trained the LaneNet with my own data. How can I test it to get a percentage number of correct detected lines (Accuracy). Only testing on single or multiple images is not enough. I hope someone can help me :)

**Comment by MaybeShewill-CV** [comment]
@LukasProgram You may see the tusimple dataset's metric figure which i think may help you:)

**Comment by LukasProgram** [comment]
Hi @MaybeShewill-CV I also used a validation set. Can you tell me something about the output? It confuses me..

![Zaznaczenie_003](https://user-images.githubusercontent.com/15108103/62104573-40039980-b2a0-11e9-89f7-c9187d0945a5.png)
![Zaznaczenie_004](https://user-images.githubusercontent.com/15108103/62104575-40039980-b2a0-11e9-8825-127f42bafd2f.png)
![Zaznaczenie_005](https://user-images.githubusercontent.com/15108103/62104576-40039980-b2a0-11e9-9438-9cc8828a6817.png)
![Zaznaczenie_006](https://user-images.githubusercontent.com/15108103/62104577-409c3000-b2a0-11e9-8e9c-63e40850f270.png)
![Zaznaczenie_007](https://user-images.githubusercontent.com/15108103/62104578-409c3000-b2a0-11e9-9548-93e172779075.png)

**Comment by MaybeShewill-CV** [comment]
@LukasProgram The model was overfitted:)

**Comment by LukasProgram** [comment]
This was my assumption too because according to the training accuracy of my model the training process could be stopped in 120k, because at this point the model detects lanes with almost 1. Am I right?
![Zaznaczenie_009](https://user-images.githubusercontent.com/15108103/62140647-a7930680-b2eb-11e9-98c7-9825ff83a5f1.png)

But how could you read this information from the validation results? can you tell how should the graph look like in the validation step (best case)?

Thx for helping

**Comment by MaybeShewill-CV** [comment]
@LukasProgram Should be almost the same as training process:)

**Comment by LukasProgram** [comment]
@MaybeShewill-CV thx
I trained it with pictures from a road simulation program.
200 training 50 validation 50 testing
What do you recommend to do in the case of overfitting?

More data? Decrease epoch number?

**Comment by MaybeShewill-CV** [comment]
@LukasProgram 1.Add more data 2. Do more data augmentation:)

**Comment by LukasProgram** [comment]
@MaybeShewill-CV 
I can add more data, but they will be similar to the other pictures because the simulation environment does not change much. I think adding more data would overfit the model even more.

How could you avoid overfitting? Would not it be better to reduce the epoch number from 200 to 100 to at least show that both models perform equally but the other requires less calculation time and no overfitting occurs? 

@MaybeShewill-CV  rlly thx for you support

**Comment by MaybeShewill-CV** [comment]
@LukasProgram 1.add more data 2.early stop:)

**Comment by longtong-system** [comment]
> Hi @MaybeShewill-CV I also used a validation set. Can you tell me something about the output? It confuses me..
> 
> ![Zaznaczenie_003](https://user-images.githubusercontent.com/15108103/62104573-40039980-b2a0-11e9-89f7-c9187d0945a5.png)
> ![Zaznaczenie_004](https://user-images.githubusercontent.com/15108103/62104575-40039980-b2a0-11e9-8825-127f42bafd2f.png)
> ![Zaznaczenie_005](https://user-images.githubusercontent.com/15108103/62104576-40039980-b2a0-11e9-9438-9cc8828a6817.png)
> ![Zaznaczenie_006](https://user-images.githubusercontent.com/15108103/62104577-409c3000-b2a0-11e9-8e9c-63e40850f270.png)
> ![Zaznaczenie_007](https://user-images.githubusercontent.com/15108103/62104578-409c3000-b2a0-11e9-9548-93e172779075.png)

hello,Can you give me a copy of the code you downloaded at that time? Because the current code has been updated, the accuracy of the call part has been deleted, so I want to refer to the code at that time, thank you
